### ISSUE TITLE
is there any tensorflow or caffe project for FCOS

### ISSUE THREAD

**Issue by YoungerGao** [tianzhi0549/FCOS#103]
thanks for your work, very interesting mind. but for the online service, the c++ api interface is needed, is there any tensorflow or caffe project for FCOS?

**Comment by tianzhi0549** [comment]
@YoungerGao There are some reimplementations based on TensorFlow. Just google them.

**Comment by YoungerGao** [comment]
@tianzhi0549 well, I will try to solve the tensorflow project, thanks

**Comment by xuannianz** [comment]
@YoungerGao I implemented FCOS in keras and tensorflow based on this official project. https://github.com/xuannianz/keras-fcos

**Comment by YoungerGao** [comment]
> @YoungerGao I implemented FCOS in keras and tensorflow based on this official project. https://github.com/xuannianz/keras-fcos

Great job!!! Have you check the difference of mAP between your tensorflow project and the official project?

**Comment by xuannianz** [comment]
@YoungerGao No, I only test on Pascal VOC. I know there's room for improvement, so I open sourced it and would like to get help from guys who are interested in it.

**Comment by YoungerGao** [comment]
> @YoungerGao No, I only test on Pascal VOC. I know there's room for improvement, so I open sourced it and would like to get help from guys who are interested in it.

Got it, thanks for your reply

**Comment by MsRabbits** [comment]
> @YoungerGao I implemented FCOS in keras and tensorflow based on this official project. https://github.com/xuannianz/keras-fcos

Thx for your project about focs with tensorflow. But there seems weird when i train my own dataset with your code: after 2 epochs, the regression_loss become nan, could you give me some advice about that?
